TarponSimulator2017/Core/Boat.cs

```diff
@@ -5,94 +5,28 @@ namespace Tarpon.Core
 {
 	public class Boat : Physics, IUpdatable
 	{
-		public enum Direction { Left = -1, Right = 1 }
 
-		public float TurnAngleSpeed { get; private set; }
-		public float TurnAngle { get; private set; }
-		public Vector2 Orientation { get; private set; }
-		
-		public Vector2 Acceleration { get; private set; }
-		public Vector2 Speed { get; private set; }
-		public Vector2 MotorPosition { get; private set; }
+		public  const float FrictionForceBoat = 0.0010f;
+		public  const float AccelerationForceBoat = 0.0005f;
+		public  const float TurnSpeedBoat = 0.0001f;
+		public  const float MaxTurnAngleBoat = 0.02f;
 
-		public Vector2 CentralPosition { get; private set; }
 
-		public const float FrictionForce = 0.0010f;
-		public const float AccelerationForce = 0.0005f;
-		public const float TurnSpeed = 0.0001f;
-		public const float MaxTurnAngle = 0.02f;
-
-		int elapsedTime; // in milliseconds
-
-		public Boat()
+		public Boat(int AbscisseDepart, int OrdonneeDepart) : base (FrictionForceBoat, AccelerationForceBoat, TurnSpeedBoat, MaxTurnAngleBoat, AbscisseDepart, OrdonneeDepart)
```

Comment:
You put some words in french.
We did not discuss this point, and I don't have strong opinions about it, but we should agree on something, preferably using only one language.

Reply:
Let's go for esperanto